x/programs/rust/wasmlanche-sdk/src/state.rs

```diff
@@ -85,3 +113,55 @@ impl Key {
         Self(bytes)
     }
 }
+
+mod host {
```

Comment:
This module was moved from the old public `host` module. It's now private and cannot be called outside of the `state` module.